app/controllers/category_transactions_controller.rb

```diff
@@ -0,0 +1,32 @@
+class CategoryTransactionsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_category, only: :new
+
+  def new
+  end
+
+  def create
+    @category_transaction = CategoryTransactions::Creator.new(category_transaction_params)
+    respond_to do |format|
+      if @category_transaction.call
+        flash[:notice] = t('transaction_create')
+      else
+        flash[:alert] = t('category_transaction_not_create')
+      end
+      format.html { redirect_to categories_path }
+      format.js
+    end
+  end
+
+  private
+
+  def category_transaction_params
```

Comment:
@marderer from the previous PRs review: we are working with `CategoryTransactionsController `, no need to seay about it once again in `params`, so method can be called `params` instead of `category_transaction_params `